components/ActivitiesList/index.js

```diff
@@ -0,0 +1,31 @@
+import ActivityCardPreview from "./ActivityCardPreview";
+import activities from "../../assets/activities";
+import styled from "styled-components";
+
+export const StyledUl = styled.ul`
+  list-style: none;
+  padding: 0;
+`;
+
+const StyledListItem = styled.li`
+  border: 1px solid black;
+  margin: 20px;
+`;
+
+export default function ActivitiesList() {
+  return (
+    <>
+      <StyledUl>
+        {activities.map((activity) => (
+          <StyledListItem key={activity.id}>
+            <ActivityCardPreview
+              title={activity.title}
+              img={activity.imageUrl}
+              categories={activity.categories}
+            />
+          </StyledListItem>
+        ))}
+      </StyledUl>
+    </>
```

Comment:
```suggestion
      <StyledUl>
        {activities.map((activity) => (
          <StyledListItem key={activity.id}>
            <ActivityCardPreview
              title={activity.title}
              img={activity.imageUrl}
              categories={activity.categories}
            />
          </StyledListItem>
        ))}
      </StyledUl>
```
you don't need the fragments here :)